tests/integration/users/test_views.py

```diff
@@ -0,0 +1,86 @@
+import json
+
+from openslides.utils.test import TestCase
+
+
+class TestWhoAmIView(TestCase):
+    url = '/users/whoami/'
+
+    def test_get_anonymous(self):
+        response = self.client.get(self.url)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content, b'{"user_id": null}')
```

Comment:
Ah If you use a Django REST Framework View for this (I would prefer) you have access to response.data.

Nevertheless you should not compare  with a bytestring. The problem is the encoding. The bystring is encoded with the (default) encoding of the test file. This is UTF-8 only because Python > 3.x uses this default now and we do not have a magic encoding line at the beginning of the file. This has nothing to do with the encoding of the data coming in. I my opinion the test should not assume things from the testing environment (here: the encoding of the test file).

My preference is using Django REST Framework View and response.data and let the framework do the rest.